angular-ngrx-scss/src/app/home/profile/profile.component.scss

```diff
@@ -11,6 +11,7 @@ main {
 }
 
 .profile-body {
+  grid-template-rows: max-content 1fr;
   .subpage {
```

Comment:
nitpick(style): spacing between this properties and the selectors is not consistent with the rest of the project. We should replicate spacing as done between lines 2-4 to keep style consistent throughout the file and project.